astropy/tests/helper.py

```diff
@@ -465,17 +464,16 @@ def assert_quantity_allclose(actual, desired, rtol=1.e-7, atol=None,
                                                                rtol, atol),
                                **kwargs)
 
-
+@deprecated("3.1","moved function to astropy.units.quantity")
```

Comment:
This is not a helpful message. I think we can just use `alternative` instead of the `message` argument, as in `alternative='astropy.units.allclose'`